teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -55,6 +56,24 @@ public class ValidatorOptions {
       arity = "1")
   private Path graffitiFile;
 
+  @Option(
+      names = {"--validators-graffiti-client-append-format"},
+      paramLabel = "<STRING>",
+      description =
+          """
+                          Appends CL and EL clients information separated with a space to user's graffiti on the Beacon Node.
+                          AUTO modes tries to provide complete version information if there is enough space for it.
+                          To disable use NONE. Available options:
+                            AUTO_END: (default) Clients info is appended after user's graffiti if any.
+                            AUTO_START: Clients info is added before user's graffiti.
```

Comment:
i think the original ticket had a definite idea about where it should be, so allowing users to choose it somewhere else is probably bad....
https://hackmd.io/@wmoBhF17RAOH2NZ5bNXJVg/BJX2c9gja

Reply:
its a long confusing thread, we may want to just verify what people are thinking on that front

Reply:
Yeah, I saw this one, mentioned it. It looks just like one of the options.
It's not a standard and not a big deal to parse it from the both sides.
User must have 100% right to put whatever he wants there so,

Reply:
Anyway I can modify it to have only START options (removing START) or mention in thread discussion that we want to give user an option to put it on any side. My personal view that END option looks better for user as graffiti belongs to him in the first place. But as I said I'm ok for any resolution.